build/SharedFxInstaller.targets

```diff
@@ -254,13 +263,51 @@
       <DebSharedFxProps>$(DebSharedFxProps);DebDependencies=$(DebSharedFxDependencies);SharedFxArchive=$(SharedFxIntermediateArchiveFilePrefix)-linux-x64.tar.gz</DebSharedFxProps>
     </PropertyGroup>
 
-    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDeb" Properties="$(CommonProps);$(DebSharedFxProps)" />
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDeb" Properties="$(CommonProps);$(DebSharedFxProps)" >
+      <Output TaskParameter="TargetOutputs" ItemName="DebsGenerated" />
+    </MSBuild>
 
     <!-- Remove Docker Image to save disk space -->
     <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_RemoveDockerImage" Properties="Image=$(Image)" />
   </Target>
 
   <Target Name="GenerateDebs" DependsOnTargets="_EnsureInstallerPrerequisites">
-    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="PackageVersion=$(PackageVersion);Image=ubuntu.14.04" />
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="PackageVersion=$(PackageVersion);Image=ubuntu.14.04" >
+      <Output TaskParameter="TargetOutputs" ItemName="DebsGenerated" />
+    </MSBuild>
+  </Target>
+
+  <Target Name="_PublishDeb">
+    <PropertyGroup>
+      <DebianRepoServer>azure-apt-cat.cloudapp.net</DebianRepoServer>
```

Comment:
Remove. We don't want to be hard-coding publish-paths anymore. Instead, add checks like this: 
https://github.com/aspnet/Universe/blob/734b3cefe8caea04b722e2886d2c64d1548e3b43/build/Publish.targets#L186-L188

Deployment settings should be in CI only.